cpp/velox/memory/VeloxMemoryManager.cc

```diff
@@ -46,7 +46,7 @@ class ListenableArbitrator : public velox::memory::MemoryArbitrator {
   }
 
   uint64_t growCapacity(velox::memory::MemoryPool* pool, uint64_t targetBytes) override {
-    std::lock_guard<std::recursive_mutex> l(mutex_);
+    // std::lock_guard<std::recursive_mutex> l(mutex_);
```

Comment:
@zhztheplayer Could you please help me figure out how to resolve this conflict? Can I just remove it?

Reply:
I remember we don't need the locks like this one anymore. Could try removing.

Reply:
Thank you, I will remove it.